SignalServiceKit/tests/Storage/TSStorageIdentityKeyStoreTests.m

```diff
@@ -20,8 +19,8 @@ @implementation TSStorageIdentityKeyStoreTests
 
 - (void)setUp {
     [super setUp];
-    
-    [[TSStorageManager sharedManager] purgeCollection:@"TSStorageManagerTrustedKeysCollection"];
+
+    [[OWSPrimaryStorage sharedManager] purgeCollection:@"OWSPrimaryStorageTrustedKeysCollection"];
```

Comment:
Wait, shouldn't this be the _symbol_ `OWSPrimaryStorageTrustedKeysCollection` whose value is `TSStorageManagerTrustedKeysCollection`?